app/models/ems_refresh/link_inventory.rb

```diff
@@ -47,7 +47,7 @@ def link_ems_inventory(ems, target, prev_relats, new_relats, disconnect = true)
     update_relats(:folders_to_hosts, prev_relats, new_relats) do |f|
       folder = instance_with_id(EmsFolder, f)
       break if folder.nil?
-      disconnect_proc = do_disconnect ? proc { |h| folder.remove_host(instance_with_id(Host, h)) } : nil
+      disconnect_proc = proc { |h| folder.remove_host(instance_with_id(Host, h)) } if do_disconnect
```

Comment:
This might actually be a change.  Previously it set the value to nil explicitly, but after it's not settings the value.  If this variable is in a binding, or this is done in a loop, then this effectively keeps the previous value instead of settings it to nil.